snorkel/classification/scorer.py

```diff
@@ -32,12 +34,14 @@ def __init__(
         if custom_metric_funcs is not None:
             self.metrics.update(custom_metric_funcs)
 
-    def score(self, golds, preds, probs):
+    def score(
+        self, golds: ArrayLike, preds: ArrayLike, probs: ArrayLike
+    ) -> Dict[str, float]:
         metric_dict = dict()
 
         for metric_name, metric in self.metrics.items():
             # Handle no examples
-            if len(golds) == 0:
+            if len(golds) == 0:  # type: ignore
```

Comment:
Which ArrayLike has no `len`?

Reply:
It doesn't know what to do with scipy.sparse.